src/imp/noop_imp.js

```diff
@@ -22,7 +29,7 @@ function NoopTracerImp() {
 }
 NoopTracerImp.prototype.setInterface = function (inf) {
 }
-NoopTracerImp.prototype.startSpan = function (nameOrFields, fields) {
```

Comment:
I think `nameOrFields` should be omitted here, right? (since this as an impl, and the wrapper calls `startSpan` with exactly one parameter every time)